deployHandler.go

```diff
@@ -66,12 +98,12 @@ func (dc *DeployCloudAppPayload) validate() ([]string, error) {
 }
 
 // NewDeployHandler creates a cloudApp controller.
-func NewDeployHandler(logger Logger, paasService PaaSService, namer Namespacer) DeployHandler {
+func NewDeployHandler(logger Logger, deployController controller.Deploy, namer Namespacer) DeployHandler {
 	return DeployHandler{
-		logger:      logger,
-		paasService: paasService,
-		namer:       namer,
-		objectNamer: rhmap.Service{},
+		logger:           logger,
+		deployController: deployController,
```

Comment:
@philbrookes It doesn't the word controller doesn't mean a http handler. The deployHandler has a deployController the deployController's Job is to control and coordinate a deploy. It knows nothing about http